Tests/External/Plugins/ASCompletion.Tests/MainForm.cs

```diff
@@ -0,0 +1,378 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+using FlashDevelop.Utilities;
+using PluginCore;
+using ScintillaNet.Configuration;
+using WeifenLuo.WinFormsUI.Docking;
+
+namespace FlashDevelop
+{
+    public class MainForm : Form, IMainForm
```

Comment:
I know all of that, I cannot change the name of the class, for some reason needed setup code checks for the type name... There are a lot of dependencies and couplings between concrete classes, plus a lot of code is mainly static, but that's not something I have intention on changing for the moment.